src/ui/widgets/HistogramWidgetUI.js

```diff
@@ -291,6 +293,18 @@ function HistogramWidgetUI(props) {
     click: clickEvent,
   };
 
+  if (data.length === 0) {
+    return (
+      <div>
+        <Alert severity='warning'>
+          <AlertTitle>No data available</AlertTitle>
+      There are no results from your data. If you are using filters, try tweaking them or zoom / pan the map to adjust the
```

Comment:
We can take a look to react-i18n.. Keeping labels in a json, and thinking about a possible internationalization of the app https://react.i18next.com/

Reply:
Aggree, but we decided to not enter into intl yet